app/view_models/admin/needs_attention_auctions_view_model.rb

```diff
@@ -12,7 +12,7 @@ def drafts_partial
   end
 
   def drafts
-    @_drafts ||= Auction.unpublished.map do |auction|
+    @_drafts ||= AuctionQuery.new.unpublished.map do |auction|
```

Comment:
Well, I think it's a bit weird that sometimes I instantiate an AuctionQuery object and sometimes I call a class method on Auction but I. defer. to. you!

Seriously though, if we do decide to change this, it should not be in this PR so let me unroll